### PR TITLE
Add explode to Typescript implementation

### PR DESCRIPTION
**Pull request by krowter**

### CODE REVIEW

File: typescript/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+dist/
```
Comment (aldy505) [inline typescript/.gitignore:2]: might as well use this https://www.toptal.com/developers/gitignore/api/node,webstorm

File: typescript/src/string/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default as explode } from "./explode";
```
Comment (aldy505) [inline typescript/src/string/index.ts:1]: bisa lebih di simplify.

di `explode.ts` langsung aja: `export const explode = ()`.
di file ini, bisa `export * from "./explode";`

**Review by ronnygunawan — COMMENTED** [review]
We need a linter. @aldy505 bisa pake rule kita ngga?

**Comment by aldy505** [comment]
> We need a linter. @aldy505 bisa pake rule kita ngga?

Bisa. Tinggal install eslint sama `@teknologi-umum/eslint-config-typescript` aja.

**Comment by aldy505** [comment]
kita merge dulu aja ya, CI sama linter bisa di setup later on

**Pull request merged**